index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#">My Logo</a>
+        <img id="menu-icon" src="images/Enabled.png" alt="" />
+      </nav>
+    </header>
+
+    <main>
+      <h1>I'm Shimels, Glad to see you!</h1>
+      <p>
+        Full-stack developer with over 3 years of experience in developing web
+        applications. Proficient in both front-end and back-end technologies,
+        including Angular, ASP.NET Core Web API, Node.js, Flask, and SQL.
+        Skilled in troubleshooting and maintaining databases and server-side
+        infrastructure. Adept at analytical thinking and problem-solving.
+        Excellent communication skills and customer focus.
+      </p>
```

Comment:
- [ ] You have done great so far :+1: But as the previous reviewer suggested, you need to stick to the Figma design as much as you can. In this section, you need to use the text provided in the Figma design. I'll suggest you update this. :smile: :wave: 
- - Your current design:
![Screenshot from 2023-07-05 15-36-54](https://github.com/shimels1/portfolio_microverse/assets/107632780/4ac02f1d-b852-4a77-8f72-07bbacd005c1)

 - - Figma design:
![Screenshot from 2023-07-05 15-36-19](https://github.com/shimels1/portfolio_microverse/assets/107632780/3bf13602-4e30-4db6-8925-a9962963e7a3)